repm/api.go

```diff
@@ -232,7 +245,10 @@ func (conn *connection) dispatchOpenTransaction(reqBytes []byte) ([]byte, error)
 		return nil, err
 	}
 
-	txID := conn.newTransaction()
+	txID, err := conn.newTransaction(req.Name, req.Args)
```

Comment:
IIUC, the name and args just weren't passed through before?

Reply:
Correct.